__tests__/app.test.js

```diff
@@ -1,15 +1,36 @@
 const pool = require('../lib/utils/pool');
```

Comment:
I would change the name of this file to `books.test.js`, since this file only tests your books routes.